backend/src/routes/v1/version.spec.ts

```diff
@@ -105,8 +105,7 @@ describe('test version routes', () => {
   //   expect(deploymentService.emailDeploymentOwnersOnVersionDeletion).not.toBeCalled()
```

Comment:
Have you checked whether this tests can now work with vitest?

Reply:
I have not.  I would rather get this vitest change merged in before trying to re-enable things that were similarly broken in Jest.

Reply:
I had them working with Jest with in my original PR, we couldn't get them working with ESM-Jest